adapters/smartrtb/smartrtb.go

```diff
@@ -31,9 +31,9 @@ type bidRequestExt struct {
 // bidExt.CreativeType values.
 const (
 	creativeTypeBanner string = "BANNER"
-	creativeTypeVideo         = "VIDEO"
-	creativeTypeNative        = "NATIVE"
-	creativeTypeAudio         = "AUDIO"
+	creativeTypeVideo  string = "VIDEO"
+	creativeTypeNative string = "NATIVE"
+	creativeTypeAudio  string = "AUDIO"
```

Comment:
This is valid Go code. Why does the linter want to change it?

Reply:
here's the linter's issue description: [SA9004 - Only the first constant has an explicit type](https://staticcheck.io/docs/checks/#SA9004).

In this case, there is no risk of confusion about the constants type since it is `string`.  But if we think this way, why do we need to mark the first constant's type as `string` - and not the consequent once?  This is inconsistent and may lead to believe that the `string` type is propagate as it is when there is no value given.  But for now I will add a `//nolint` and restore the existing code.

Reply:
reverted to the original version, added `// nolint: staticcheck // staticcheck SA9004: only the first constant in this group has an explicit type`